test_schemas/en/test_theme_business.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Not sure we need this schema, your code above uses an else, so when I did it I just picked a non social one. I guess there is a bigger question, are we going to have business as a theme, up to now its kind of the default

Reply:
I test existing non social one, the test business theme schema removed! Thanks.